src/sagemaker/jumpstart/model.py

```diff
@@ -312,6 +314,27 @@ def _is_valid_model_id_hook():
 
         super(JumpStartModel, self).__init__(**model_init_kwargs.to_kwargs_dict())
 
+    def retrieve_default_payload(self) -> JumpStartSerializablePayload:
+        """Returns default payload associated with the model.
+
+        Payload can be directly used with the `sagemaker.predictor.Predictor.predict(...)` function.
```

Comment:
this is customer facing: please add a `Raises` section.

Reply:
if `payloads.retrieve_options` can throw, please port over the appropriate item of its `Raises` section.

Related question: do you need a way to propagate the `tolerate_vulnerable` and `tolerate_deprecated` flags?